ntcore/src/main/native/include/networktables/Topic.h

```diff
@@ -0,0 +1,384 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <stdint.h>
+
+#include <string>
+#include <string_view>
+#include <utility>
+#include <vector>
+
+#include "networktables/NetworkTableType.h"
+#include "ntcore_c.h"
+#include "ntcore_cpp.h"
+
+namespace wpi {
+class json;
+}  // namespace wpi
+
+namespace nt {
+
+class GenericEntry;
+class GenericPublisher;
+class GenericSubscriber;
+class NetworkTableInstance;
+
+/** NetworkTables Topic. */
+class Topic {
+ public:
+  Topic() = default;
+  explicit Topic(NT_Topic handle) : m_handle{handle} {}
+
+  /**
+   * Determines if the native handle is valid.
+   *
+   * @return True if the native handle is valid, false otherwise.
+   */
+  explicit operator bool() const { return m_handle != 0; }
```

Comment:
Don't we want the implicit conversion available here?

Reply:
Operator bool should nearly always be explicit.